vllm_spyre/v1/worker/spyre_model_runner.py

```diff
@@ -375,6 +386,14 @@ def prepare_model_input(
         else:
             return self._prepare_decode(scheduler_output.scheduled_cached_reqs)
 
+    def get_supported_tasks(self) -> tuple[SupportedTask, ...]:
+        tasks = list[SupportedTask]()
+
+        if "generate" in self.model_config.supported_tasks:
+            tasks.extend(["generate"])
```

Comment:
Ideally we would want this to be coming from the model directly:
```
if is_text_generation_model(model):
            supported_tasks.append("generate")
```
but `SpyreCausalLM` doesn't seem to support it atm

```
type(self.model)
<class 'vllm_spyre.model_executor.model_loader.spyre.SpyreCausalLM'>

is_text_generation_model(self.model)
False

```